Binary files netpbm-9.9/pnm/fiasco/.config.h.swp and netpbm-9.9.new/pnm/fiasco/.config.h.swp differ
diff -urN netpbm-9.9/pnm/fiasco/config.h netpbm-9.9.new/pnm/fiasco/config.h
--- netpbm-9.9/pnm/fiasco/config.h	Mon Feb 12 17:34:43 2001
+++ netpbm-9.9.new/pnm/fiasco/config.h	Mon Feb 12 17:29:00 2001
@@ -17,7 +17,7 @@
 #define STDC_HEADERS 1
 
 /* Define if you can safely include both <sys/time.h> and <time.h>.  */
-#define TIME_WITH_SYS_TIME 1
+/* #undef TIME_WITH_SYS_TIME 1 */
 
 /* Define if your processor stores words with the most significant
    byte first (like Motorola and SPARC, unlike Intel and VAX).  */
@@ -74,7 +74,7 @@
 #define HAVE_STRING_H 1
 
 /* Define if you have the <sys/time.h> header file.  */
-#define HAVE_SYS_TIME_H 1
+/* #undefi HAVE_SYS_TIME_H 1 */
 
 /* Define if you have the <unistd.h> header file.  */
 #define HAVE_UNISTD_H 1
